test/components/NavBar/__snapshots__/OidcLoginComponent.test.jsx.snap

```diff
@@ -33,6 +33,7 @@ exports[`renders correctly 1`] = `
     class="MuiBox-root css-120gsus"
   />
   <div>
+    http://localhost:3000/PASS/
```

Comment:
We now pass the redirect URL in explicitly, causing it to render correctly in the test (it wasn't before, and I didn't notice).